docs/hub/spacy.md

```diff
@@ -24,7 +24,7 @@ All models on the Hub come up with useful features
 All `spaCy` models from the Hub can be directly installed using pip install.
 
 ```bash
-pip install https://huggingface.co/spacy/en_core_web_sm/resolve/main/en_core_web_sm-any-py3-none-any.whl
+pip install "en_core_web_sm @ https://huggingface.co/spacy/en_core_web_sm/resolve/main/en_core_web_sm-any-py3-none-any.whl"
```

Comment:
LGTM! You might want to update https://github.com/huggingface/huggingface.js/blob/main/packages/tasks/src/model-libraries-snippets.ts#L698 as well